<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ZipScout</string>
  </property> 
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="searchLayout">
      <item>
       <widget class="QLabel" name="searchLabel">
        <property name="text">
         <string>Search word:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="searchWordEdit">
        <property name="text">
         <string>secret</string>
        </property>
        <property name="placeholderText">
         <string>Enter word to search</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
	  <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="summaryTab">
       <attribute name="title">
        <string>Summary</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="fileLabel">
          <property name="text">
           <string>Analysing archive:: N/A</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="progressBar">
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="logTextEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="selectFileButton">
            <property name="text">
             <string>Choose ZIP-file</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="cancelButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Cancel</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="clearButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Clear</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="saveButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Save results</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="filesTab">
       <attribute name="title">
        <string>Files</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QTableView" name="filesTableView">
          <property name="selectionMode">
           <enum>QAbstractItemView::NoSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <property name="showGrid">
           <bool>true</bool>
          </property>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Include</string>
           </property>
           <property name="clickable">
            <bool>true</bool>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Filename</string>
           </property>
           <property name="clickable">
            <bool>true</bool>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Size</string>
           </property>
           <property name="clickable">
            <bool>true</bool>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Modified</string>
           </property>
           <property name="clickable">
            <bool>true</bool>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>